specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/examples/CosmosDBRestorableSqlResourceList.json

```diff
@@ -0,0 +1,35 @@
+{
+  "parameters": {
+    "api-version": "2020-06-01-preview",
+    "subscriptionId": "2296c272-5d55-40d9-bc05-4d56dc2d7588",
+    "location": "WestUS",
+    "instanceId": "d9b26648-2f53-4541-b3d8-3044f4f9810d",
+    "restoreLocation": "WestUS",
+    "restoreTimestampInUtc": "10/13/2020 4:56"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "databaseName": "Database1",
+            "collectionNames": [
```

Comment:
>collectionNames [](start = 13, length = 15)

nit: containerNames

Reply:
The restore api takes in a generic DatabaseRestoreResource object that's not API specific, and we are using the same object definition for both SQL and MongoDB here, so we don't have distinction between collection/container.

This will change once we update the restore api to takes in API specific parameters